src/store/store.js

```diff
@@ -63,6 +63,8 @@ export const store = new Vuex.Store({
     BNPLAccess: [...admin],
     CreditCheckAccess: [...admin, 34, 16, 21, 22],
     CoordinatorAccess: [...admin, 32],
+    CoordinatorAndVCRAccess: [...admin, 32, 22],
```

Comment:
`rather than do this, do this.auth('CoordinatorAccess') || this.auth('DVAAccess')` in the NewOrderAmortization

Reply:
So delete this

Reply:
DVAAccess will give others access in which only Admin, Coordinators and VCR should be able to change status, I asked babajide and he said VCR is 22, that's why I did it like that.

Reply:
Check
` DVAAccess: [...admin, 13, 16, 21, 22, 23, 26, 32, 34],
`
This has taken care of it.  we only add access according to roles

Reply:
It has been taken care of. Just do this
 this.auth('CoordinatorAccess') || this.auth('DVAAccess')

Reply:
Okay